doc/backends/azure-blob-storage.md

```diff
@@ -0,0 +1,49 @@
+### Goofys with Azure Blob Storage
+
+Goofys is already can work with Azure Blob Storage out of the box.
+To achive successful mount of azblob container to your fs, you need to follow next simple steps:
+
+- Get account name and blob container name from your Azure Portal
+- Get the Access key for your account from Azure Portal like described [here](https://help.bittitan.com/hc/en-us/articles/115008109327-How-do-I-get-an-access-key-for-Azure-Blob-storage- "here").
+- You can put account and key values to the config file like this:
+
+```ini
```

Comment:
I'd probably change it to the same format as existing instruction for AWS:

```ini
$ cat ~/.azure/config
storage]
account = "myblobstorage"
key = "MY-STORAGE-KEY"
$ $GOPATH/bin/goofys wasb://container <mountpoint>
$ $GOPATH/bin/goofys wasb://container:prefix <mountpoint> # if you only want to mount objects under a prefix
```

Reply:
Does it already work? I will check anyway